frontend/src/pages/content/ProcessMethodology.tsx

```diff
@@ -0,0 +1,84 @@
+import { useTranslation } from "next-i18next";
+import Link from "next/link";
+import { Button, Grid, Icon } from "@trussworks/react-uswds";
+
+import { ExternalRoutes } from "src/constants/routes";
+import ContentLayout from "src/components/ContentLayout";
+
+const ProcessMethodology = () => {
+  const { t } = useTranslation("common", { keyPrefix: "Process" });
+
+  return (
+    <div className="padding-top-4 bg-gray-5">
+      <Grid tabletLg={{ col: 6 }} className="grid-container">
+        <strong className="tablet-lg:font-sans-lg tablet-lg:margin-bottom-05">
```

Comment:
This isn't showing up left aligned in desktop view in my browser:
![image](https://github.com/HHS/simpler-grants-gov/assets/25465464/7732938f-314d-4517-8297-1f1c4219cb07)